### PR TITLE
[Sections Mapping] Fix: Any malformed URL in meta tags result in complete import failure

### PR DESCRIPTION
**Pull request by catalan-adobe**
fixes https://git.corp.adobe.com/Granite/esaas-demo-tool/issues/93.

### CODE REVIEW

File: js/sections-mapping/import/sections-mapping.import.js [hunk 1]
```diff
@@ -242,6 +239,13 @@ export default {
         }
       });
 
+      // do not create metadata for now for crosswalk as it may break when converting to JCR.
```
Comment (Danialkafeel) [inline js/sections-mapping/import/sections-mapping.import.js:242]: Can you please briefly explain the need of this change?

File: js/sections-mapping/import/sections-mapping.import.js [hunk 2]
```diff
@@ -273,6 +270,9 @@ export default {
       cleanUpAttributes(el);
       el.querySelectorAll('*').forEach((e) => cleanUpAttributes(e));
 
+      // default helix-importer rules
+      WebImporter.rules.adjustImageUrls(el, params.originalURL, params.originalURL);
+
```
Comment (catalan-adobe) [inline js/sections-mapping/import/sections-mapping.import.js:275]: @Danialkafeel, this is the actual fix of the issue. The idea is to adjust the URLs of all the images.

File: js/sections-mapping/import/sections-mapping.import.js (hunk 1 above)
Comment (catalan-adobe) [reply js/sections-mapping/import/sections-mapping.import.js:242]: This is not the actual fix. This is another adjustment to get the metadata created only once per fragment.

File: js/sections-mapping/import/sections-mapping.import.js (hunk 2 above)
Comment (Danialkafeel) [reply js/sections-mapping/import/sections-mapping.import.js:275]: Got it thanks

**Review by Danialkafeel — APPROVED** [review]
LGTM. Given that it is tested...

**Pull request merged**